packages/auth-sveltekit/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@edgedb/auth-sveltekit",
+  "description": "Helper library to integrate the EdgeDB Auth extension with Sveltekit.",
+  "version": "0.1.0-alpha.1",
+  "author": "EdgeDB <info@edgedb.com>",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/edgedb/edgedb-js.git",
+    "directory": "packages/auth-svelte"
+  },
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "build": "tsc --project tsconfig.json"
+  },
+  "devDependencies": {
+    "@types/node": "^20.8.4",
+    "edgedb": "^1.3.6",
+    "typescript": "^5.2.2"
+  },
+  "peerDependencies": {
+    "edgedb": "^1.3.6"
+  },
+  "dependencies": {
+    "@edgedb/auth-core": "^0.1.0-beta.1",
+    "@sveltejs/kit": "^2.0.0",
+    "svelte": "^4.2.7",
+    "vite": "^5.0.3",
```

Comment:
`svelte` and `vite` should probably be dev dependencies, as they're only needed to make sveltekit's postinstall script happy, and I don't think we want to define deps here that could conflict with the user's installed versions.

Reply:
Also not sure if `@sveltejs/kit` should be a peer dependency (+ dev dep) instead? @scotttrinh

Reply:
Yeah, I that's how we structured the others: https://github.com/edgedb/edgedb-js/blob/316a398f9dbb413a880440086df7b5bebc7e68b5/packages/auth-nextjs/package.json#L23-L35. I think the only dependency should be on `@edgedb/auth-core` itself.